detection-rules/office_remote_html_template.yml

```
@@ -25,7 +25,7 @@ source: |
               // detect files without an extension
               or .file_type in ("docx")
           )
-          and any(beta.oletools(.).relationships, iregex_search(.target, ".*html!"))
+          and any(beta.oletools(.).relationships, iregex_search(.target, ".*html?!"))
```

no need for leading `.*` when doing a search.  doesn't matter for performance because MQL should strip them out under the hood.
```suggestion
          and any(beta.oletools(.).relationships, iregex_search(.target, "html?!"))
```